gt-scheduler/website
# Text for Hidden Courses

### Summary

Resolves #188 



Adds text at the bottom of the calendar to inform users of courses that will not appear on the calendar

### Checklist

- [x] The bottom text is displayed with all TBA and weekend sections. Each section must include the course name + the section code.
- [x] When a user exports the calendar via downloading the image, the text should also display in the exported image.
- [x] The text should be available in appropriate ratio (text should not be cut off)
- [x] The text should be wrapped (moved to the next line) or scrollable automatically according to screen size
- [x] Mobile view works.


### How to Test

Add a course that is not on the Atlanta campus or has a Saturday meeting time, and the text of the course name and section will appear at the bottom of the calendar. When you 'download image', the text will also appear at the bottom of the calendar.
